Quelques modifications sur la Table Of Contents et style LaTeX

t00sh opened the pull request:
* Augmentation de la profondeur
    * Passage des commandes dans les titres en \emph{} qui donne un meilleur
      résultat visuel

Je la met en WIP, car il faut peut être pour être cohérent dans notre plan : 
- rajouter une subsection sur le "persistant mode" 
- découper davantage le chapitre 1 en subsection
- revoir le plan du chapitre 3, voir rajouter des sections

Commit:
Quelques modifications sur la Table Of Contents

    * Augmentation de la profondeur
    * Passage des commandes dans les titres en \emph{} qui donne un meilleur
      résultat visuel

Pamplemousse commented:
Ah mince, j'avais pas fait attention au fait que c'était `[WIP]`.
Pourquoi ce statut ?

t00sh commented:
Pour le commentaire sur les choses à modifier.

Mais on peut merger, et modifier plus tard.

Commit:
Réglage des marges

Commit:
Quelques modifications du plan.

    - Ajout d'une partie sur le persistent mode
    - Ajout d'une partie sur les autres versions d'AFL
    - Modifications de quelques subsubsections en subsections
    - Ajout du chapter3.tex dans les dépendances du Makefile

t00sh commented:
Quelques modifications du plan :
    
        - Ajout d'une partie sur le persistent mode (pas rédigée)
        - Ajout d'une partie sur les autres versions d'AFL (pas rédigée)
        - Modifications de quelques subsubsections en subsections
        - Ajout du chapter3.tex dans les dépendances du Makefile

Pamplemousse commented on report/chapter1.tex:
```diff
@@ -287,3 +287,7 @@ \subsubsection{"fork server"}
   \item{} Et dans \url{https://github.com/mirrorer/afl/blob/master/afl-as.h}, avec les fonctions \lstinline{__afl_forkserver}, \lstinline{__afl_fork_resume}, \lstinline{__afl_fork_wait_loop} et la variable \lstinline{__afl_fork_pid}, le code injecté dans le programme cible.
 \end{itemize}
+
+\subsection{Le ``persistent mode''}\label{persistent-mode}
```
Ne pas le faire apparaitre tant qu'il sera pas rédigé : je préfère garder `master` comme une branche "production ready".
En plus, on a #15 qui référence cela.

Pamplemousse commented on report/chapter3.tex:
```diff
@@ -71,3 +69,7 @@ \subsubsection{Instrumentation par AFL}
 Ainsi, lorsqu'un nouveau bloc est atteint, le parent est averti, puis le cache est mis à jour, et chaque fils peut alors bénéficier de la traduction qui vient d'être effectuée.
 Ainsi, le mode QEMU est deux à cinq fois plus lent que l'utilisation normale d'AFL.
+
+\section{Autres versions de AFL}
```
Même remarque.
